src/core/Chip/BaseChip/BaseChip.baseStyles.tsx

```diff
@@ -21,15 +21,15 @@ export const baseChipBaseStyles = (theme: SuomifiTheme) => css`
     padding: ${theme.spacing.insetXxs} ${theme.spacing.insetL};
     color: ${theme.colors.whiteBase};
     background: ${theme.colors.highlightBase};
-    max-height: 28px;
+    max-height: 100%;
     display: inline-block;
 
     & .fi-chip--content {
-      display: inline-block;
-      max-width: 270px;
-      white-space: nowrap;
+      display: block;
+      max-width: 100%;
```

Comment:
max-width is not needed here. Comes also from base css class

Reply:
And to clarify, the `max-width: 100%;` comes from the span's resets

https://github.com/vrk-kpa/suomifi-ui-components/blob/15eaf2f030efe9b0b61f9dacf47d88f2469d6c4c/src/reset/HtmlSpan/HtmlSpan.tsx#L11-L19

Reply:
Fixed.